charts/aad-pod-identity/README.md

```diff
@@ -103,11 +103,40 @@ kubectl delete crd azureidentitybindings.aadpodidentity.k8s.io
 kubectl delete crd azurepodidentityexceptions.aadpodidentity.k8s.io
 ```
 
-## Upgrading to a New Major Chart Version
+## Upgrade guide
+
+### Upgrading from chart version 1.5.5
+
+1.5.5 helm chart had introduced 2 labels which could possibly change with chart upgrade -
+
+```yaml
+      app.kubernetes.io/managed-by: Helm
+      helm.sh/chart: aad-pod-identity-1.5.5
+```
+
+This has been fixed in chart version `1.5.6` to prevent any issues with future upgrades of helm chart. For upgrading from 1.5.5 to any new chart version, a suggested workaround is editing the nmi and mic manifests to remove those 2 labels from `selector.matchLabels`
```

Comment:
```suggestion
This has been fixed in chart version `1.5.6` to prevent any issues with future upgrades of helm chart. For upgrading from 1.5.5 to any new chart version, a suggested workaround is editing the nmi and mic manifests to remove those 2 labels from `selector.matchLabels`:
```